docs/getting_started/tutorial.ipynb

```diff
@@ -18,7 +18,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
Be careful about pushing changes to a notebook which aren't substantive changes. I always try to "Restart kernel and clear all outputs" before saving the notebook. That usually avoids execution_count changes and such.

Reply:
https://github.com/metagraph-dev/metagraph/pull/120/commits/d303ca095a4410154834df02e3de7e9bb871da3e removes these accidental changes.